loops-and-arrays.js

```diff
@@ -41,6 +57,13 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+console.log(squares)
+
+for (let i = 0; i < squares.length; i ++){
+    squares[i] = 'Het volume van ' + squares[i] + ' is ' + squares[i] * squares[i] * squares[i];
+}
+
+    console.log(squares)
 
```

Comment:
hey man, het ziet er allemaal prima. ik heb het hier iets anders gedaan en ik heb ook gewerkt met de backtick. 
for (let i = 0; i < squares.length; i++) {
    const volume = squares[i] * squares[i] * squares[i];
    squares[i] = `Het volume van ${squares[i]} is ${volume}`;
}.
ik vind dat zelf lekkerder werken maar neemt niet weg dat je het gewoon goed hebt gedaan natuurlijk:)